roles/core/log_client/vars/RedHat_7.yml

```diff
@@ -0,0 +1,7 @@
+---
+log_server_packages_to_install:
```

Comment:
The variable must be `log_client_packages_to_install`, the role log_client does not use `log_server_*` variables.
Also, note that vars/RedHat_X.yml will override vars/main.yml, so you need to add the requested package **rsyslog** here.

Reply:
done... sorry for this fix to be so complex to fix for me :-)